app/api/projects/[id]/route.ts

```diff
@@ -0,0 +1,51 @@
+import { deleteProject, getProject, updateProject } from "../repository";
+import { NextRequest, NextResponse } from "next/server";
```

Comment:
Changed the folder structure from `project` to `projects` as using plural is a rest api standard I am used to but happy to swap it back if anyone disagrees!

Reply:
i like plural.